### PR TITLE
Add Nov blog post

### PR DESCRIPTION
**Pull request by dholbach**
@staceypotter @scottrigby @stefanprodan @makkes @souleb @darkowlzz

### CODE REVIEW

**Comment by dholbach** [comment]
→ https://deploy-preview-610--fluxcd.netlify.app/blog/2021/11/november-2021-update/

File: content/en/blog/2021-10-29-october-update.md [hunk 1]
```diff
@@ -0,0 +1,329 @@
+---
+author: dholbach
+date: 2021-10-20 20:30:00+00:00
```
Comment (scottrigby) [inline content/en/blog/2021-10-29-october-update.md:3]:
```suggestion
date: 2021-10-29 20:30:00+00:00
```

**Review by scottrigby — APPROVED** [review]
Apart from updating the date, lgtm!

**Review by kingdonb — DISMISSED** [review]
I only found two issues. I would clarify with the Flagger folks whether `nginx-ingress` should read as `ingress-nginx`, (looking over the PR, I think it should, but I'm not 100% sure myself) and, corrected the spelling of my name. 👍 LGTM

File: content/en/blog/2021-10-29-october-update.md [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+v1beta2
+API](https://github.com/fluxcd/flux2/discussions/1916)
+guide.
+
+### Flagger 1.15 is out
+
+We are blessed and fortunate to have our own progressive delivery
+solution within the Flux project. Its 1.15 release brings support for
+NGINX ingress canary metrics (you will need nginx-ingress v1.0.2 at
+least).
```
[the remark this answers is not in the document]
Comment (dholbach) [reply content/en/blog/2021-10-29-october-update.md:79]: That's where I copied the info from.

[the remark this answers is not in the document]
Comment (dholbach) [reply content/en/blog/2021-10-29-october-update.md:79]: @stefanprodan ?

**Review by makkes — APPROVED** [review]
Great post, so many news

**Pull request merged**